src/egon/data/datasets/gas_areas.py

```diff
@@ -191,16 +191,21 @@ def create_voronoi(scn_name, carrier):
     if len(buses) == 0:
         return
 
-    buses["x"] = buses.geometry.x
-    buses["y"] = buses.geometry.y
     # generate voronois
-    gdf = get_voronoi_geodataframe(buses, boundary.geometry.iloc[0])
-    # set scn_name
+    if len(buses) == 1:
```

Comment:
This function is only used for status2019 scenario, right? 
I'm a bit wondering why two cases are needed. Isn't there always just one bus in this scenario?

Reply:
Ah okay, sorry for my confusion. 
But it would be nice to have a comment on why these two cases could appear.

Reply:
Thank you!

Reply:
I think that would be useful. I included that explanation in b5c65def00aaed6ae37dcb62fe0a6419e2b2d5a2.